step-controller-plugins/step-controller-plugins-raw-measurement/pom.xml

```diff
@@ -1,48 +1,48 @@
-<!--
-  Copyright (C) 2020, exense GmbH
-   
-  This file is part of STEP
-   
-  STEP is free software: you can redistribute it and/or modify
-  it under the terms of the GNU Affero General Public License as published by
-  the Free Software Foundation, either version 3 of the License, or
-  (at your option) any later version.
-   
-  STEP is distributed in the hope that it will be useful,
-  but WITHOUT ANY WARRANTY; without even the implied warranty of
-  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  GNU Affero General Public License for more details.
-   
-  You should have received a copy of the GNU Affero General Public License
-  along with STEP.  If not, see <http://www.gnu.org/licenses/>.
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-
-	<artifactId>step-controller-plugins-rtm</artifactId>
-	<packaging>jar</packaging>
-
-	<parent>
-		<groupId>ch.exense.step</groupId>
-		<artifactId>step-controller-plugins</artifactId>
-		<version>0.0.0-SNAPSHOT</version>
-	</parent>
-
-	<dependencies>
-		<dependency>
-			<groupId>ch.exense.step</groupId>
-			<artifactId>step-controller-measurement</artifactId>
-			<version>${project.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>ch.exense.rtm</groupId>
-			<artifactId>rtm</artifactId>
-			<version>${rtm.version}</version>
-		</dependency>
-	</dependencies>
-
-	<build>
-		<plugins>
-		</plugins>
-	</build>
-</project>
+<!--
+  Copyright (C) 2020, exense GmbH
+
```

Comment:
no reformatting of existing files should be done in a PR, keep only relevant changes